src/blockdiag/imagedraw/textfolder.py

```diff
@@ -13,38 +13,40 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-import re
-
 from blockdiag.utils import XY, Box, Size
 
 
-def splitlabel(string):
+def splitlabel(text):
     """Split text to lines as generator.
-       Every line will be stripped.
-       If text includes characters "\n", treat as line separator.
+
+    Every line will be stripped. If text includes characters "\n\n", treat
+    as line separator. Ignore '\n' to allow line wrapping.
     """
-    string = re.sub(r'^\s*', '', string)
-    string = re.sub(r'\s*$', '', string)
-    string = re.sub(r'\xa5', '\\\\', string)
-    string = re.sub('(\\\\){2}', '\x00', string)
-    string = re.sub('\\\\n', '\n', string)
-    for line in string.splitlines():
-        yield re.sub('\x00', '\\\\', line).strip()
```

Comment:
Why did you remove these code? I can't accept such breaking change.

Reply:
This was mostly accidental. However, I've no idea what this is doing. Could you help me understand so I can adjust my change to suit?

Reply:
I've groked what most of it's doing, but I have one question

    string = re.sub(r'^\s*', '', string)

Removes leading whitespace. Handled by `strip()`

    string = re.sub(r'\s*$', '', string)

Removes trailing whitespace. Handled by `strip()`

    string = re.sub(r'\xa5', '\\\\', string)

Replaces the Yen symbol with `\\\\`. I don't know why this is happening. Do you?

    string = re.sub('(\\\\){2}', '\x00', string)

Replaces `\\\\` with `\x00` (the NULL character). Again, I don't know why this is done.

    string = re.sub('\\\\n', '\n', string)

Replaces an escaped newline with an actual newline. Handled by `splitlines()`?

    yield re.sub('\x00', '\\\\', line).strip()

Replaces the null character added previously with `\\\\`. Again, why?